src/assets/docs/getting-started-with-dockstore.md

```diff
@@ -10,11 +10,12 @@ Your link to GitHub is established on login and you will then be prompted to lin
 
 ![Link accounts](../assets/images/docs/linking1.png)
 
-Linking a supported image repository service (e.g. Quay.io) will automatically trigger a synchronization order to retrieve information about the account's tools
+<!-- Currently UI2 does not perform and automatic refresh all tools -->
```

Comment:
Hmmm, please make an issue for this bug and add instructions on how to trigger a refresh manually here.

Reply:
Issue created.  The next section already talks about refresh all tools.

Reply:
I see it, but its quite buried which doesn't really flow well and could be confusing. 
Let's add instructions on how to trigger a refresh manually here and take it out when this bug is fixed in a future hotfix